micrometer-spring-legacy/src/test/java/io/micrometer/spring/advice/TimedMethodInterceptorTest.java

```diff
@@ -0,0 +1,239 @@
+package io.micrometer.spring.advice;
+
+import io.micrometer.core.annotation.Timed;
+import io.micrometer.core.aop.TimedAspect;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.Tags;
+import io.micrometer.core.instrument.simple.SimpleMeterRegistry;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.EnableAspectJAutoProxy;
+import org.springframework.context.annotation.Import;
+import org.springframework.stereotype.Service;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class TimedMethodInterceptorTest {
```

Comment:
Should this be public as per JUnit 4 conventions? As for sake of consistency with other tests within this module?